engine/src/main/java/org/destinationsol/ResolutionProvider.java

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
This class the same as previous.

Reply:
Sorry, previous comment probably didn't save. I was writing that you should consider moving the Resolution class, as well as this one, to the subpackage `menu`, as these are just utility classes for that screen. I am however open to another opinions from anyone, be it you or another reviewer.